examples/device/cdc_msc_freertos/src/main.c

```diff
@@ -97,9 +97,11 @@ int main(void)
   // skip starting scheduler (and return) for ESP32-S2
 #if CFG_TUSB_MCU != OPT_MCU_ESP32S2
   vTaskStartScheduler();
+#if CFG_TUSB_MCU != OPT_MCU_RX63X
```

Comment:
Okay, I understood to keep the `return 0` statement. So, I should no need to update for this part, right?

Since `NVIC_SystemReset()` is not available for RX family, I added `#if CFG_TUSB_MCU != OPT_MCU_RX63X`. If you mind the ifdef, I will add a dummy macro or function to a somewhere header to eliminate ifdef.

Reply:
hah, I just realized that rx63n doesn't have `NVIC_SystemReset()` , we can actually remove it since scheduler start does not return at all. I will remove it :)

Reply:
Thanks! :smile: I didn't notice until you mentioned it.